app/views/embeddable/multiple_choices/_show_choice_likert.html.haml

```diff
@@ -0,0 +1,7 @@
+%div.likert{:style => "display: table-cell; text-align: center; width:150px; vertical-align: bottom;"}
+  = "#{choice.choice}"
+  %br
+  - if embeddable.multi_answer
+    = check_box_tag 'embeddable_multiple_choice_answer[answers][]', choice.id, embeddable.answers.include?(choice), :id => "embeddable_multiple_choice_answer_answers_#{choice.id}", :data => {:button_id => check_button_string}
+  - else
+    = form.radio_button :answers, choice.id, :data => {:button_id => check_button_string}, :checked => (embeddable.answers.include?(choice))
```

Comment:
Because the 4 lines above are duplicated with the `_show_choice.html.haml` partial. It would be better if you combined these two partials into one. The style of the top level div can be done just looking at the inherit layout class from the .choice-container. So the only thing that needs to be conditionalized is the position of the `choice.choice`.